src/sentry/search/events/filter.py

```diff
@@ -1129,3 +1140,29 @@ def _issue_filter_converter(self, search_filter: SearchFilter) -> Optional[Where
                 SearchValue(filter_values if search_filter.is_in_filter else filter_values[0]),
             )
         )
+
+    def format_aggregate_filter(
+        self,
+        aggregate_filter: AggregateFilter,
+    ) -> Optional[WhereType]:
+        name = aggregate_filter.key.name
+        value = aggregate_filter.value.value
+
+        if name in self.params.get("aliases", {}):
+            raise NotImplementedError(
+                f"aggregate alias: {name} not implemented in snql filter parsing yet"
+            )
+
+        value = (
+            int(to_timestamp(value))
+            if isinstance(value, datetime) and name != "timestamp"
+            else value
+        )
```

Comment:
honestly not sure if this is necessary but I copied it from [here](https://github.com/getsentry/sentry/blob/master/src/sentry/search/events/filter.py#L113-L115)